content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -693,6 +699,302 @@ kubectl apply -f my-crontab.yaml
 crontab "my-new-cron-object" created
 ```
 
+## Validation rules
+
+{{< feature-state state="alpha" for_k8s_version="v1.23" >}}
+
+Validation rules are in alpha since 1.23 and validate custom resources when the
+`CustomResourceValidationExpressions` [feature
+gate](/docs/reference/command-line-tools-reference/feature-gates/) enabled and the schema is a
```

Comment:
nit: is enabled

Reply:
Does this mean we allow x-kubernetes-validations in non-structural schemas (of old pre-v1 CRDs), but wouldn't validate?

Reply:
Rephrased.